client/components/theme-collection/style.scss

```diff
@@ -1,31 +1,93 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.theme-collection__container {
+	padding-bottom: 32px;
+
+	&:last-child {
+		padding-bottom: 0;
+
+		@include break-xlarge {
+			padding-bottom: 32px;
+		}
+	}
+
+	.theme-collection__list-wrapper {
+		padding-right: 32px;
+
+		&.swiper-wrapper {
+			overflow-y: clip;
+		}
+	}
+
+	.theme-card__info-options {
+		display: none;
+
+		@include break-small {
+			display: inherit;
+		}
+	}
+
+	.theme-card__image-container {
+		border-radius: 4px;
+		box-shadow: none;
+	}
+}
+
 .theme-collection__meta {
 	display: flex;
 	justify-content: space-between;
 }
 
 .theme-collection__headings {
-	flex-basis: 45%;
+	flex-basis: 80%;
+
+	@include break-small {
+		flex-basis: 45%;
+	}
 }
 
 .theme-collection__title {
-	font-size: 1.75rem;
 	line-height: 1;
+	font-size: 1.25rem;
+	font-weight: 400;
+	color: var(--gray-gray-100, #101517);
+
+	@include break-small {
+		font-size: 1.75rem;
+	}
 }
 
 .theme-collection__description * {
+	color: var(--gray-gray-70, #3c434a);
 	margin-bottom: 0;
+
+	@include break-small {
+		font-size: 1rem;
+	}
 }
 
 .theme-collection__list-item.swiper-slide {
-	/* TODO: Needs a better solution. Testing on more screen widths etc */
-	width: 32%;
+	width: 80%;
```

Comment:
I don't hold a strong opinion, but it could be better to use swipers built-in `breakpoints` and `sliderPerView` settings: https://swiperjs.com/swiper-api#param-breakpoints

Reply:
You can use floats for `slidesPerView` and then you don't need the `padding-right: 32px;` on the -wrapper

Reply:
I prefer just needing to change one file instead of two. I did see how you implemented it on the PR you had opened but I think that it makes more sense to not distribute the presentation styles across multiple components.

Reply:
In the end, I'm going to need to use the breakpoints. When having the `slidesPerView` set to `auto` we can't calculate the width of the contained elements and we can't fine control the position which leads to mismatching elements. So I'm using the `breakpoints` + `slidesPerView`.